.github/workflows/.trivyignore

```diff
@@ -1,4 +1,10 @@
 # Date: June 16, 2023
 # Issue: snappy-java high issue, https://www.cvedetails.com/cve/CVE-2023-34453/
 # Solution: Spring needs to update its version
-CVE-2023-34455
\ No newline at end of file
+CVE-2023-34455
```

Comment:
Is this still relevant or can we upgrade spring?

Reply:
We use latest version of spring-kafka, which uses 3.4 kafka by default, but according to [this](https://spring.io/projects/spring-kafka), kafka 3.5 should be compatible with spring. 

Manually upgraded kafka in the meantime, made a note in trivyignore.